recipes-sca/checkbashism-native/checkbashism-native_2.23.5.bb

```diff
@@ -35,3 +35,5 @@ do_install() {
 }
 
 FILES:${PN} += "${bindir}"
+
+RDEPENDS:${PN}:class-nativesdk += "nativesdk-perl"
```

Comment:
That's not needed, as kirkstone lacks the support for SDK tools iirc

Reply:
I'll remove the RDEPENDS.